### PR TITLE
updates README and add LICENCE and CONTRIBUTING

### PR DESCRIPTION
**Pull request by gregLibert**

### CODE REVIEW

**Comment by gregLibert** [comment]
@qdrn cc

**Comment by gregLibert** [comment]
> It looks nice. I would just mention docs.massa.net.

Readme updated

**Review by Thykof — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -1,32 +1,42 @@
...
-Fill the issue number. The issue would probably be a meta issue. Sub-issues would be issue to implement
-the standard in proper repositories.
+The [DApps <> Wallet Provider Communication standard](wallet/dapps-communication.md) defines a common interface for communication between Massa-based decentralized applications (DApps) and wallet providers.
+This standard aims to simplify the process of integrating Massa-based DApps with various wallet providers, making it easier for end-users to access and use these applications.
 
-> _NOTE:_ this [VSCode extension](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint)
-can help you to format markdown files.
+## Getting Started
+
+To get started with the Massa Standard Definition project, please refer to the documentation in the `docs` directory. This documentation includes instructions for contributing to the project and implementing the various standards defined here.
```
[the remark this answers is not in the document]
Comment (gregLibert) [reply README.md]: fixed

File: README.md [hunk 2]
```diff
@@ -1,32 +1,42 @@
...
-```
+## DApps <> Wallet Provider Communication
 
-Fill the issue number. The issue would probably be a meta issue. Sub-issues would be issue to implement
-the standard in proper repositories.
+The [DApps <> Wallet Provider Communication standard](wallet/dapps-communication.md) defines a common interface for communication between Massa-based decentralized applications (DApps) and wallet providers.
+This standard aims to simplify the process of integrating Massa-based DApps with various wallet providers, making it easier for end-users to access and use these applications.
 
-> _NOTE:_ this [VSCode extension](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint)
-can help you to format markdown files.
```
[the remark this answers is not in the document]
Comment (gregLibert) [reply README.md:32]: Why not adding it in a CI ?

[the remark this answers is not in the document]
Comment (gregLibert) [reply README.md:32]: (i would like people to focus on standard and not necessary on code practices ...)

File: README.md [hunk 3]
```diff
@@ -1,32 +1,42 @@
...
+## Getting Started
+
+To get started with the Massa Standard Definition project, please refer to the documentation in the `docs` directory. This documentation includes instructions for contributing to the project and implementing the various standards defined here.
+
+## License
+
+This project is licensed under the Apache 2.0 license. For more information, please refer to the `LICENSE` file.
```
[the remark this answers is not in the document]
Comment (gregLibert) [reply README.md]: fixed

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Thykof) [reply README.md:32]: new issue created https://github.com/massalabs/massa-standards/issues/39

**Review by Thykof — APPROVED** [review]

**Pull request merged**